docs/language/configs.md

```diff
@@ -157,13 +158,16 @@ content text {
 ```
 
 {{< hint warning >}}
-Fabric limits which environment variables are exposed inside the templates. By default, only the
-variables that match `FABRIC_*` name glob pattern are available under `env` namespace and in the
-evaluation context. The glob pattern is configured using `expose_env_vars_with_pattern` argument
-in [Global configuration]({{< ref "configs.md#global-configuration">}}).
 
-Name-based filtering minimizes the risk of unintentionally leaking sensitive environment variables to
-untrustworthy plugins. It's recommended to use restrictive glob patterns as much as possible.
+To minimize the risk of unintentionally leaking sensitive environment variables to untrustworthy
+plugins, Fabric limits which environment variables are exposed in the evaluation context.
+
```

Comment:
I'd change the wording here a little bit, "untrustworthy plugins" sounds quite bad when all of the plugins are written by us 😉 

```suggestion
In accordance to the principle of least privelege, Fabric limits which environment variables are exposed to plugins in the evaluation context.

```

Reply:
good point!